### DISCUSSION TITLE
shell . --login

### DISCUSSION THREAD

**Discussion by m-delc** [kovidgoyal/kitty#5490] [General]
Endeavour OS.

I need to run kitty with --login. Searching `kitty.conf` for login makes this seem easy enough to accomplish but whatever I type into the config makes kitty crash. I've tried

`shell . --login`
`shell /usr/bin/kitty --login`
`shell /bin/kitty --login`

None of these work, and so I'm out of ideas. How can I accomplish this?

**Comment by page-down** [comment]
kitty is not a shell, but a terminal emulator.

> I need to run kitty with --login. ...

What you really need is to provide the `--login` option when running the shell as a command line program in kitty.

You should configure the option for the shell you are using, for example `shell /usr/local/bin/bash -l`.

https://wiki.archlinux.org/title/command-line_shell

**Reply by m-delc** [reply]
of course.... obvious newb here. Thanks for your help.

**Comment by kovidgoyal** [comment]
shell /bin/bash --login
